src/app/projectView.hook.js

```diff
@@ -53,12 +52,26 @@ export default (
         .then(() => {
           ProjectLoadingEmitter.emit(FINISHED_LOADING_PROJECT_EVENT);
         })
-        .catch(() => {
+        .catch(targetState => {
+          let stateService = trans.router.stateService;
+
+          if (targetState) {
```

Comment:
would be good to add a basic check here that this is actually a `TargetState` and not an `Error`.

something like `if (targetState && targetState.withParams) {`